lib/extensions/ar_order.rb

```diff
@@ -0,0 +1,21 @@
+# This monkey patches rails to support proper joins
+# This is a current PR for 6.1: https://github.com/rails/rails/pull/36531
```

Comment:
The wording here confused me.  The code below is the original 6.1.   I believe this comment needs to be applied when you're monkey patching it.  Right?

Reply:
moved